tock/employees/utils.py

```diff
@@ -0,0 +1,68 @@
+import datetime
+
+from django.db.models import Sum
+
+from hours.models import TimecardObject, ReportingPeriod
+
+def get_timecardobjects(user):
+    queryset = TimecardObject.objects.filter(
+        user=user,
+        timecard__submitted=True
+    )
+    queryset = queryset.exclude(
+        project__name='Out of Office'
+    )
+    return queryset
```

Comment:
This method might be better served as a `TimecardObject` manager method instead; see how to [add new manager methods and bind custom managers here](https://docs.djangoproject.com/en/1.10/topics/db/managers/#adding-extra-manager-methods) in the Django docs.